index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Abel portfolio</title>
+  </head>
+  <body>
+    <!-- Navbar -->
+    <nav class="nav">
+      <logo>
+        <p class="logo">Abel Tsegaye</p>
+      </logo>
+      <img class="mobileMenu" src="images\IconMenu.png" alt="Menu" />
+    </nav>
+
+    <!-- Headline -->
+    <headline class="headline">
+      <h1>Hey There.<br />I'm Abel</h1>
+      <h2>I'm a Software Developer</h2>
+      <p>
+        I can help you build a product , feature or website Look through some of
+        my work and experience! If you like what you see and have a project you
+        need coded, don’t hestiate to contact me.
+      </p>
+      <socials class="socials">
+        <img class="socialLink" src="images\GitHub.png" alt="Github" />
+        <img class="socialLink" src="images\LinkedIn.png" alt="LinkedIn" />
+        <img class="socialLink" src="images\Anglelist.png" alt="Anglelist" />
+        <img class="socialLink" src="images\Twitter.png" alt="Twitter" />
+        <img class="socialLink" src="images\Medium.png" alt="Medium" />
+      </socials>
+      <hedlineImage>
+        <img class="bgImage" src="images\Illustration.png" />
+      </hedlineImage>
+    </headline>
```

Comment:
- [ ] Kindly make sure to wrap the headline container into the `section` tag instead of using `headline`. It represents a generic standalone section of the document, so use it. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info.